AlfrescoDocumentPickerFileProvider/AFPDataManager.m

```diff
@@ -209,6 +220,20 @@ - (void)updateMetadataForIdentifier:(NSFileProviderItemIdentifier)itemIdentifier
     }
 }
 
+- (AFPItemMetadata *)localFilesItem
+{
+    AFPItemMetadata *localFiles = nil;
+    RLMRealm *realm = [self realm];
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"identifier = %@", [AFPItemIdentifier itemIdentifierForSuffix:nil andAccountIdentifier:nil]];
+    RLMResults<AFPItemMetadata *> *results = [AFPItemMetadata objectsInRealm:realm withPredicate:predicate];
+    if(results && results.count > 0)
```

Comment:
check for "results.count" is sufficient, no need to check for nullity of "results" var